### ISSUE TITLE
Pygdal 3.3.0

### ISSUE THREAD

**Issue by marcoaurelioneto** [nextgis/pygdal#63]
Any forecast to make pygdal version 3.3.0 available?

**Comment by samcom12** [comment]
Following for an update.

**Comment by marcoaurelioneto** [comment]
Ok, waiting. Thanks!

**Comment by bdowning** [comment]
Addressed in PR #64 

For reference adding a new GDAL release is very easy, and takes only a couple minutes:

```
$ ./import 3.3.0
$ vi VERSIONS # add version
$ vi README.md # edit valid versions range
$ git push
```

**Comment by dezhin** [comment]
Yes, adding a new GDAL release is simple. I've merged @bdowning's PR and pushed 3.3.0 version to PyPi. But 3.3.0 version requires additional testing because of dropping Python 2 support.

Could anybody confirm that it is working correctly with GDAL 3.3.0? Currently, I don't have a suitable environment to test.
